notebooks/official/prediction/get_started_with_vertex_private_endpoints.ipynb

```diff
@@ -742,7 +714,7 @@
       "source": [
         "### Create VPC peering for default network\n",
         "\n",
-        "For simplicity, we setup VPC peering to the default network. You can create a different network for your project.\n",
+        "For simplicity, you setup VPC peering to the default network. You can create a different network for your project.\n",
         "\n",
         "If you setup VPC peering with any other network, make sure that the network already exists and that your VM is running on that network."
```

Comment:
change from noun to verb: "If you set up VPC peering..."

Reply:
Did the required changes